content/applications/websites/livechat/responses.rst

```diff
@@ -1,48 +1,178 @@
-of your most common questions and comments.
+In the Odoo *Live Chat* application, *Commands* allow the user to perform specific actions both
+inside the chat window, and through other Odoo applications. The *Live Chat* app also includes
+*Canned Responses*. These are customized, pre-configured substitutions that allow users to replace
+shortcut entries in place of longer, well-thought out responses to some of the most common
+questions and comments.
 
-Use commands
-============
+Both *commands* and *canned responses* save time, and allow users to maintain a level of
+consistency throughout their conversations.
 
-Commands are shortcuts that do specific actions within the chat window:
+Execute a command
+=================
 
-#. **/help**: shows a help message.
-#. **/helpdesk**: creates a helpdesk ticket.
-#. **/helpdesk_search**: searches for a helpdesk ticket.
-#. **/history**: shows the last 15 visited pages.
-#. **/lead**: creates a new lead.
-#. **/leave**: leaves the channel.
+Live chat *commands* are keywords that triggers pre-configured actions. When a live chat *operator*
+is participating in a conversation with a customer or visitor, they can execute a command by typing
+`/` followed by the command.
+
+Commands, and the resulting actions, will only be visible in the conversation window for a live
+chat operator. A customer will not see any commands entered on their end of the chat.
+
+  .. image:: responses/live-chat-responses-ticket-link.png
+     :align: center
+     :alt: View of the chat window with a helpdesk ticket created in Odoo Live Chat.
+
+More information about each available command can be found below.
+
+Help
+----
+
+If an operator types `/help` in the chat window, an informative message that includes the potential
+entry types an operator can make is displayed.
+
+.. image:: responses/live-chat-responses-help.png
+   :align: center
+   :alt: View of the message generated from using the /help command in Odoo Live Chat.
+
+.. seealso::
+
+  - :doc:`/applications/productivity/discuss/overview/get_started`
+  - :doc:`/applications/productivity/discuss/overview/team_communication`
+
+Helpdesk & Helpdesk search
+--------------------------
+
+The `/helpdesk` and `/helpdesk_search` commands allow operators to both create :guilabel:`Helpdesk`
+tickets directly from a conversation, and search through existing tickets by keyword or ticket
+number.
 
 .. important::
-   | - For *helpdesk tickets*: make sure the application is installed on your database and the
-     option *Live Chat* under :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams`
-     is enabled.
-   | - For *leads*: the *CRM* application must be installed on your database.
+   - The `/helpdesk` and `/helpdesk_search` commands can only be used if the :guilabel:`Helpdesk`
+     app has been installed, and :guilabel:`Live Chat` has been activated on a :guilabel:`Helpdesk
```

Comment:
Italicize _Helpdesk_, _Live Chat_, and _Helpdesk Team_ before the active instruction begins